src/app/rundown-view/components/unfocused/unfocused.component.html

```diff
@@ -1 +1 @@
-<span class="label">{{ outOfFocusMessage }}</span>
+<span class="label" [style.margin-top.px]="getCenterPositionForLabel()">{{ outOfFocusMessage }}</span>
```

Comment:
See other comment for explanation for this change 😄 

```suggestion
<span class="label" [style.margin-top.px]="marginTop">{{ outOfFocusMessage }}</span>
```

Reply:
It has been changed :-)